### PR TITLE
fix blade ascension not giving you the ring of blades

### LINKED ISSUE #85635: Heretic Blade Ascension doesn't work
## Issue Summary

The heretic's blade ascension doesn't work properly, not adding the proper self-regenerating blades.

## Reproduction:

Ascend as blade

Notice blades don't regenerate properly

### LINKED ISSUE #87344: Blade Heretic still has no post-ascend blades & no cooldown on blade shield ability
### Client Version:

n/a

### Issue Summary:

1. Blade Heretic STILL does not gain blades after ascending.
2. Malestrom of Silver or whatever has no cooldown - it can be cast continously after expending all blades.

Also for the latter if you get hit while the silver blades are appearing, the rest of them don't appear, I dunno if thats intended or not

Placing together because they are related to the recent blade rework PR.

### Round ID:

[24053](https://scrubby.melonmesa.com/round/24053)

### Test Merge Information:

n/a

### Reproduction Steps:

1. get maelstrom of silver
2. notice it has no cd
3. get ascension
4. notice no blades spawn

### PR DESCRIPTION
**Pull request by Xander3359**
## About The Pull Request
fix blade ascension not giving you the ring of blades
## Why It's Good For The Game
Bugfix
Closes https://github.com/tgstation/tgstation/issues/87344
Closes https://github.com/tgstation/tgstation/issues/85635
## Changelog
:cl:
fix: fix blade ascension not giving you the ring of blades
/:cl:

### CODE REVIEW

**Comment by SmArtKar** [comment]
Duplicate of a fix in #87432 I believe?

File: code/modules/antagonists/heretic/knowledge/blade_lore.dm [hunk 1]
```diff
@@ -444,7 +444,7 @@
 	)
 	ADD_TRAIT(user, TRAIT_NEVER_WOUNDED, name)
 	RegisterSignal(user, COMSIG_HERETIC_BLADE_ATTACK, PROC_REF(on_eldritch_blade))
-	user.apply_status_effect(/datum/status_effect/protective_blades/recharging, null, 8, 30, 0.25 SECONDS, 1 MINUTES)
+	user.apply_status_effect(/datum/status_effect/protective_blades/recharging, null, 8, 30, 0.25 SECONDS, 1 MINUTES, /obj/projectile/floating_blade)
```
Comment (MrMelbert) [inline code/modules/antagonists/heretic/knowledge/blade_lore.dm:447]: I am pretty certain you should not be passing a projectile to spawn in place of an `/obj/effect`

**Review by MrMelbert — COMMENTED** [review]
In fact this is the real issue

base type

image

subtype

image

Notice blade type is 6 for base, and 7 for subtype

Whomever added the blade type arg didn't update the subtypes correctly so they're not in the right order. All you really have to do is realign them to fix it

File: code/modules/antagonists/heretic/knowledge/blade_lore.dm (hunk 1 above)
Comment (Xander3359) [reply code/modules/antagonists/heretic/knowledge/blade_lore.dm:447]: It always has been

**Review by MrMelbert — APPROVED** [review]

**Pull request merged**